src/patternfly/components/Nav/nav.scss

```diff
@@ -1,6 +1,7 @@
 .pf-c-nav {
   --pf-c-nav--Width: #{pf-size-prem(290px)};
   --pf-c-nav--Transition: var(--pf-global--Transition);
+  --pf-c-nav__item--m-expanded__toggle--child--Transform: rotate(90deg);
```

Comment:
@mcoker Since there is no icon class here should I use `child` in the selector name, or should we create an icon class to be consistent.

Reply:
An icon class would be great. The data-list would need that update, too. Let's be sure and leave the `*` as the selector so the icon still rotates for people who don't have the new markup, then create a follow up issue to target the icon class in the selector in a breaking change release.